### ISSUE TITLE
Question about preparing VOCwithSBD datasets

### ISSUE THREAD

**Issue by yasuobinggan** [Haochen-Wang409/U2PL#112]
Sorry to bother, I'm not quite sure how the VOCwithSBD dataset was prepared.

Are the steps as follows?

1. Download VOCtrainval_11-May-2012.tar, and extract "VOC2012 folder" from it
2. Download SegmentationClassAug.zip, and extract "SegmentationClassAug folder" from it
3. Put the "SegmentationClassAug folder" into "VOC2012 folder", then we can use the code to train based the "VOC2012 folder" directly(without other processes)

Thank you!

**Comment by Haochen-Wang409** [comment]
Did you encounter any problems when reproducing the results after doing so?

**Comment by yasuobinggan** [comment]
I'm new and I want to confirm how the VOCwithSBD dataset is prepared to prevent mistakes.

Now I am experimenting with 2 X 3090, using the "1464" setting, and the "batchsize" parameter is set to 4. The current experimental results are as follows：
![3)`MN7%O%0KF)6TO95TAW9T](https://user-images.githubusercontent.com/30293157/225212997-13527a01-9d1d-4b8a-b3d7-82a5efe01525.jpg)
![N8IXHUKL DNJ_PZ6UCQK$22](https://user-images.githubusercontent.com/30293157/225213026-4c1ecb32-dcff-435b-87a0-84bea017934d.png)
I noticed the learning rate issue mentioned, I will try to reproduce the results on 2 X 3090 in the future.

**Comment by Haochen-Wang409** [comment]
Provided results are as expected. It seems that you have successfully reproduced this work :)

**Comment by yasuobinggan** [comment]
Thank you!
